src/static-legacy/js/functions.js

```diff
@@ -1,559 +1,73 @@
-                linkElement = this;
-                dropMenu();
-            }
-        }
-        else {
-            if(!dropping) {
-                hideMenu();
-            }
-        }
-        return false;
-    });
-    $("#subnav nav").mouseenter(function() {
-        window.clearTimeout(hidemenu);
-        window.clearTimeout(dropdown);
-        var hovering = "#" + $(this).attr("id");
-    }).mouseleave(function() {
-        var hovering = "";
-        hidemenu = window.setTimeout(hideMenu,500);
-    });
-    $(".inf input").keyup(function() {
-      if ($(this).val()) {
-        $(this).addClass("filled");
-      }
-      else {
-        $(this).removeClass("filled");
-      }
-    });
-    $(".inf textarea").keyup(function() {
-      if ($(this).val()) {
-        $(this).addClass("filled");
-      }
-      else {
-        $(this).removeClass("filled");
-      }
-    });
-    $(".signup").submit(function() {
-        var form = $(this);     form.children("fieldset").children("p:last-child").children("button").attr("disabled","disabled");
-        $.ajax({
-            type: "POST",
-            url: $(this).attr("action"),
-            data: form.serialize(),
-            complete: function(req, status_msg) {
-                if(status_msg){
-                    form.children("fieldset").hide();
-                    var thanks;
-                    if(form.attr("data-thanks")) {
-                        thanks = form.attr("data-thanks");
-                    }
-                    else {
-                        thanks = "Thanks, we’ll be in touch.";
-                    }
-                    form.append("<p>"+thanks+"</p>");
-                } else {
-                    form.children("fieldset").hide();
-                    form.append("<p>Something went wrong. Please try again later.</p>");
-                }
-            }
-        });
-        return false;
-    });
+    // Remove styles from feature images
     $("a > img").each(function() {
         $(this).parent().addClass("noStyles");
     });
-    $(".mini a").mouseup(function(){
-        var link = $(this).attr("href");
-        link = link.replace("http://","/");
-        link = link.replace("http://","/");
-        var tracker = "/v/topshare"+link;
-        _gaq.push(['_trackPageview',tracker]); });
-    $(".botshare a").mouseup(function(){
-        var link = $(this).attr("href");
-        link = link.replace("http://","/");
-        link = link.replace("http://","/");
-        var tracker = "/v/botshare"+link;
-        _gaq.push(['_trackPageview',tracker]);
-    });
-    $('iframe').each(function() {
-        var url = $(this).attr("src")
-        if ( url.indexOf('youtube') !== -1 ) {
-            var char = "?";
-            if ( url.indexOf("?") !== -1 ) {
-                char = "&";
-            }
-            $(this).attr("src",url+char+"wmode=transparent");
-        }
-    });
+
+    // Add pdf class to pdf links
     $('a[href$="pdf"]').each(function() {
```

Comment:
This is for the legacy code in `/budget/strategic-plan/interactive/` so may not follow our updated design. @sebworks do you know?